### PR TITLE
Make width/height default to 1, remove UA "guessing"/faking geometry

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/w3c/pointerevents/issues/65

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -244,11 +244,11 @@
 				</dd>
 				<dt>readonly attribute double width</dt>
 				<dd>
-					The width (magnitude on the X axis), in CSS pixels (see [[CSS21]]), of the <a>contact geometry</a> of the pointer.  This value MAY be updated on each event for a given pointer. For devices which have a contact geometry but the actual geometry is not reported by the hardware, a default value SHOULD be provided by the <a>user agent</a> to approximate the geometry typical of that pointer type. Otherwise, the value MUST be 0.
+					The width (magnitude on the X axis), in CSS pixels (see [[CSS21]]), of the <a>contact geometry</a> of the pointer.  This value MAY be updated on each event for a given pointer. For inputs that typically lack contact geometry (such as a traditional mouse or stylus), and in cases where the actual geometry of the input is not detected by the hardware, the <a>user agent</a> MUST return a default value of 1.
```
Comment (mustaqahmed) [inline index.html]: May we possibly omit "stylus"? Stylus "contact area" could be considered equivalent to "having a geometry".

**Comment by teddink** [comment]
Change looks good to me. I am fine either with or without the reference to stylus.

File: index.html (hunk 1 above)
Comment (patrickhlauke) [inline index.html]: good point...in hindsight, i completely agree that a stylus has a contact geometry. going to make the change and force-push

**Comment by RByers** [comment]
LGTM

**Comment by patrickhlauke** [comment]
Merging, as no objections. https://www.w3.org/2016/05/25-pointerevents-minutes.html

**Pull request merged**